sky/backends/provision_lib.py

```diff
@@ -0,0 +1,24 @@
+""""Cloud-neutral VM provision library.
+
+This module controls the lifecycle of VMs - creation, setup, termination.
+"""
+from sky import provision
+
+
+def teardown_cluster(cloud_name: str, region: str, cluster_name: str,
+                     terminate: bool) -> None:
+    """Deleting or stopping a cluster."""
+    if terminate:
+        provision.terminate_instances(cloud_name, region, cluster_name)
+        # TODO(suquark): enable this line in later PRs about the
+        #  new provisioner
+        # metadata_utils.remove_cluster_metadata(cluster_name)
+    else:
+        provision.stop_instances(cloud_name, region, cluster_name)
```

Comment:
Question: do we really need this module, as it only serves as a router based on `terminate` argument, and it does not seem to be a complicated enough function to have this router. Should we directly let the `backend` call the `provision.terminate_instances` and `provision.stop_instances`?